src/utils/loginurls.ts

```diff
@@ -0,0 +1,26 @@
+const isBrowser = () => typeof window !== 'undefined';
+
+const deriv_api_url = 'deriv.com';
```

Comment:
can we move all of these to `utils/constants`?
thanks :pray:

Reply:
Done